common/service.sh

```diff
@@ -2,8 +2,8 @@
 # More info in the main Magisk thread
 
 if [ $API -ge 28 ] && [ $FULL ]; then
-  pm enable "com.google.android.apps.wellbeing/com.google.android.apps.wellbeing.autodnd.ui.AutoDndGesturesSettingsActivity"
   am force-stop "com.google.android.apps.wellbeing"
+  pm enable "com.google.android.apps.wellbeing"
```

Comment:
Why are we doing this?

Reply:
i just removed the autodnd line

pm enable "com.google.android.apps.wellbeing/com.google.android.apps.wellbeing.autodnd.ui.AutoDndGesturesSettingsActivity"

Reply:
But the whole thing is enabled already